### ISSUE TITLE
make a ui design of explore page for mobile view

### ISSUE THREAD

**Issue by Aadarsh805** [Aman-Mandal/groovy#43]
basic layout where we keep player fixed on bottom and other things scrollable

**Comment by deven-533** [comment]
Can you assign me this issue Sir.

**Comment by MarianCekan** [comment]
Hello, is this issue still available please?

**Comment by ARNAV1233** [comment]
@Aadarsh805  I want to contribute to the issue

**Comment by Aman-Mandal** [comment]
> Hello, is this issue still available please?
> 

You wanna work on design? Go ahead !! 

Steps : 
1. Fork and clone it
2. Install dependencies
3. Create a new folder namely "Design"
4. Create a new sub-folder called "Mobile- Explore_page"
5. Add a Figma/image file of the design u created
6. Push it and create a PR

**Comment by ARNAV1233** [comment]
Can I do this issue as well

**Comment by Aman-Mandal** [comment]
> Can I do this issue as well
 
I already assigned it @MarianCekan but If you want to work on design, you can work on the other pages, I will create an issue for it!!

Works?

**Comment by ARNAV1233** [comment]
thanks

**Comment by Aman-Mandal** [comment]
> Hello, is this issue still available please?

I have created a design folder, make sure you add your design inside it

**Comment by MarianCekan** [comment]
Hello, I was trying to find the issue where I can integrate some kind of data from already existing part of code so I will have to understand the code already written. For example to build FE for some page in your app. So I'll unassign myself from this issue.

By any chance don't you have any kind of issue I described above? (preferably labled #good first issue) We can move to private chat If you prefer

**Comment by Aman-Mandal** [comment]
> Hello, I was trying to find the issue where I can integrate some kind of data from already existing part of code so I will have to understand the code already written. For example to build FE for some page in your app. So I'll unassign myself from this issue.
> 
> By any chance don't you have any kind of issue I described above? (preferably labled #good first issue) We can move to private chat If you prefer

Heyy, No problem!! Right now we aren't having any FE part but in 1-2 days we will start working on other pages of our application, you can contribute to that!! And yeah..you can text me anytime on Twitter (given in the Readme) ..Thankyouu!!

**Comment by Rohan-Hazari** [comment]
Hey can I also try this?

**Comment by Aadarsh805** [comment]
> Hey can I also try this?
yes Rohan u can, try to make it somewhat like how Spotify do it for the player part, else you can just make it just like our desktop design

**Comment by Rohan-Hazari** [comment]
Hey, can I get some more time it's my first time trying designing in Figma so its taking time is that okay?

**Comment by Aman-Mandal** [comment]
> Hey, can I get some more time it's my first time trying designing in Figma so its taking time is that okay? 

Yeah sure !! Take your time..

Just keep us updated that's all

**Comment by peterintech** [comment]
Hi @Rohan-Hazari  can you send the link to the designs, the file is not opening so I know what am working with

**Comment by Rohan-Hazari** [comment]
@iampitrus sorry for delay
https://www.figma.com/file/lMqNIfH6Vu6bAy75acxIvS/Untitled?node-id=0%3A1

**Comment by Aadarsh805** [comment]
> @iampitrus sorry for delay https://www.figma.com/file/lMqNIfH6Vu6bAy75acxIvS/Untitled?node-id=0%3A1

these are better than what we have now, we could definitely make sidebar like that, maybe in the next update, great work

**Comment by peterintech** [comment]
Alright thanks

**Comment by Rohan-Hazari** [comment]
> > @iampitrus sorry for delay https://www.figma.com/file/lMqNIfH6Vu6bAy75acxIvS/Untitled?node-id=0%3A1
> 
> these are better than what we have now, we could definitely make sidebar like that, maybe in the next update, great work

Glad you liked it 😁
